app/helpers/cases_helper.rb

```diff
@@ -51,6 +51,7 @@ def action_button_for(event)
               class: 'button',
               method: 'post'
     when :assign_responder
+      return '' if @case.type_of_offender_sar?
```

Comment:
We use `.offender_sar?` in `_case_status.html.slim` as a replacement for `.type_of_offender_sar?` should we use it here for consistency as well?

Reply:
I don't think that's where we ended up... 🤔 sorry for the confusion

My original proposal was to use `.offender_sar?` in that sense but we changed it back to mean Offender::Standard

any place you see it it means it - any place you see `.type_of_offender_sar?` it means either standard or complaint

The above is intentional to hide the button that would show up because of the new event in the state machine config